assets/js/atomic/blocks/product-elements/product-image-gallery/style.scss

```diff
@@ -0,0 +1,3 @@
+.woocommerce .wp-block-woocommerce-product-image-gallery div.images {
+	width: unset !important;
```

Comment:
Do we need to use `!important` here?

I typically try to avoid this, if at all possible. It seems like increasing the specificity just a little bit would do the trick:

```css
.woocommerce .wp-block-woocommerce-product-image-gallery .woocommerce-product-gallery.images {
    width: unset;
}
```

I'm also wondering about the choice to use `unset` as opposed to `initial` (which is how `unset` behaves here on a non-inherited property), `auto`, or even `100%`. I'm not mentioning it because I think it's wrong, I'm really just curious. 🙂

Reply:
Thank you for reviewing this, you're right, we can avoid using !important if we increase the specificity of the css rule. 

> I'm also wondering about the choice to use unset as opposed to initial (which is how unset behaves here on a non-inherited property), auto, or even 100%. I'm not mentioning it because I think it's wrong, I'm really just curious. 🙂

Well, I think that I just put the first value that worked and solved the issue, but now that you mentioned `initial` would be much better. I just committed a new version with your suggestions 🙌

Reply:
Thanks @thealexandrelara!

And to clarify, I wasn't implying that `unset` was necessarily wrong. I was really more curious if there was a particular reason for going with that option.

IMO, there are tradeoffs – `unset` can be a little confusing because its behavior depends on if the element is an inherited or non-inherited property. On the other hand, it does show up as `unset` in the dev tools where `initial` seems to display the initial value. I typically go with `initial` because I don't have to think about the inheritance type.

Anyhow, thanks for updating. Everything LGTM and tests as described! 🚀